### DISCUSSION TITLE
postgres_changes on both insert and update at once

### DISCUSSION THREAD

**Discussion by pierroo** [supabase/supabase#23270] [Feature Requests]
I know we can subscribe to "*" events for postgres_changes; however this also contains the DELETE event which has no filter and is triggered regardless so completely useless.

for that reason I was wondering, is there anyway to subscribe to INSERT and UPDATE within the same subscriber instead of creating two listeners for the sake of simplicity?

**Comment by GaryAustin1** [comment]
![image](https://github.com/supabase/supabase/assets/54564956/264357d0-9dd4-400d-b4e8-e2bba140d4bf)
No that is not a current feature.

**Reply by pierroo** [reply]
I know it's not currently a feature, but that's why I posted it in "feature request", I would love to have it :)

**Reply by GaryAustin1** [reply]
Just clarifying as sometimes posts don't get classified correctly and you asked like you were not sure...
"
![image](https://github.com/supabase/supabase/assets/54564956/a9653c82-68b9-4ecc-9898-b4544415b279)

**Reply by pierroo** [reply]
true; I somehow tried to sneak a question in case I missed it in the docs, otherwise it was a feature request :)
not sure if these requests are checked from time to time, hopefully they are!
